test/e2e/opentelemetry/client-trace-metadata/client-trace-metadata.test.ts

```diff
@@ -37,6 +37,12 @@ describe('clientTraceMetadata', () => {
     expect(firstLoadSpanIdContent).not.toBe(secondLoadSpanIdContent)
   })
 
+  it('should only insert the client trace metadata once', async () => {
+    const html = await next.render('/suspense')
+    const matches = html.match(/meta name="my-test-key-1"/g)
```

Comment:
was this supposed to match `client-inserted-key`?

Reply:
No it doesn't matter for client-inserted-key, it's because we use the client trace metadata, it didn't went through line 83 `return ''`. So it's just for creating the test case here.